lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -172,9 +175,20 @@ data NetworkLayer m block = NetworkLayer
         :: Coin -- Stake to consider for rewards
         -> m StakePoolsSummary
 
-    , getAccountBalance
+    , getCachedAccountBalance
         :: RewardAccount
-        -> ExceptT ErrGetAccountBalance m Coin
+        -> m Coin
+        -- ^ Return the cached reward balance of an account.
+        --
+        -- If there is no cached value, it will return `Coin 0`, and add the
+        -- account to the internal set of observed account, such that it will be
+        -- fetched later.
+
+    , fetchAccountBalances
```

Comment:
occasionally I think that  `fetchRewardAccountBalances` and `getCachedRewardAccountBalance` would be better here  in spite of being long...

Reply:
👍 fair -> d81089b8bb205af2f082d0704eac7410aba1c541